install/resources/kafka/Makefile

```
@@ -0,0 +1,14 @@
+.PHONY: create/namespace
+create/namespace:
+	@echo "create namespace: $(KAFKA_CLUSTER_NAMESPACE)"
+	@sed 's/<NAMESPACE>/$(KAFKA_CLUSTER_NAMESPACE)/g' ./namespace.yaml | cat | oc apply -f -
```

Would in this case be enough to use `oc new-project ...`?

---

In this case no, as oc new-project disallows any new ns with the prefix `openshift` to prevent conflicts with system ns's

---

I also prefer namespace.yaml because we can apply it and don't need to ignore the error if it already exists.